src/extensions/extension-discovery.ts

```diff
@@ -393,7 +393,14 @@ export class ExtensionDiscovery extends Singleton {
 
     for (const extension of userExtensions) {
       if ((await fse.pathExists(extension.manifestPath)) === false) {
-        await this.installPackage(extension.absolutePath);
+        try {
+          await this.installPackage(extension.absolutePath);
+        } catch (error) {
+          const message = error.message || error || "unknown error";
```

Comment:
In the error case shouldn't you remove this extension from the list of userExtensions?

Reply:
I am pretty sure that won't help very much because we watch that folder anyway so it will be discovered again.

I would like to change how this whole discovery/installation part works, but not here.